src/ktopt_planning_context.cpp

```diff
@@ -162,10 +132,12 @@ void KTOptPlanningContext::solve(planning_interface::MotionPlanResponse& res)
     setJointVelocities(vel_val, active_joints, *waypoint);
     res.trajectory->addSuffixWayPoint(waypoint, time_step);
 
-    // also try and visualise on drake's visualizer
-    diagram_context_->SetTime(t);
     plant_->SetPositions(plant_context_, pos_val);
-    visualizer_->ForcedPublish(*visualizer_context_);
+    auto& vis_context = visualizer_->GetMyContextFromRoot(*diagram_context_);
+    visualizer_->ForcedPublish(vis_context);
+
+    // Without these sleeps, the visualizer won't give you time to load your browser
+    std::this_thread::sleep_for(std::chrono::milliseconds(500));
```

Comment:
this pause is definitely excessive, and hopefully not needed for much longer than this PR lol